### PR TITLE
Week 1 assignments

### PR DESCRIPTION
**Pull request by abhinavsingh06**

### CODE REVIEW

File: Week-1/Exercise1.2/index.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      <div class="player-image-container">
+        <img src="https://vif1g.csb.app/src/assets/cattyboard.jpg" alt="Cattyboard hot 100" />
+      </div>
+      <div class="player-content">
+        <p>If It Ain't Love</p>
+        <p>Catson Derulo</p>
+      </div>
+    </section>
+    <section class="control-container">
+      <ul class="player-control">
```
Comment (harsh-pesto) [inline Week-1/Exercise1.2/index.html]: remove UL and use flex-box

Comment (harsh-pesto) [reply Week-1/Exercise1.2/index.html]: remove li and to use div

File: Week-1/Exercise1.2/stylesheets/index.css [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
```
Comment (harsh-pesto) [inline Week-1/Exercise1.2/stylesheets/index.css]: remove all 86 lines and replace with *

File: Week-1/Exercise1.2/stylesheets/index.css [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+}
+
+.image-container {
+  height: 12rem;
+  width: 12rem;
+  margin-right: 5rem;
+}
+
+img {
+  height: 100%;
```
Comment (harsh-pesto) [inline Week-1/Exercise1.2/stylesheets/index.css:54]: bad way of doing this as you are declaring global styles for img tag , which is anti pratice. Target specific element and then override img size

**Comment by harsh-pesto** [comment]
image
play all button is completely off

**Review by harsh-pesto — COMMENTED** [review]
image
The buttons are not alligned

File: Week-1/Exercise1.1.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
+# What is a browser?
```
Comment (harsh-pesto) [inline Week-1/Exercise1.1.md:1]: Where is the lucid chart , theory is ok but it needs to be pictorial format

**Review by harsh-pesto — APPROVED** [review]
Work on the provided comments approving for moving to next week

**Review by harsh-pesto — APPROVED** [review]

File: Week-1/Exercise1.2/stylesheets/index.css (hunk 3 above)
Comment (abhinavsingh06) [reply Week-1/Exercise1.2/stylesheets/index.css:54]: @harsh-pesto I am using a container to wrap the  tag which is `image-container` while this globla property is to ensure that img always take the size of the container.
```
.image-container {
  height: 12rem;
  width: 12rem;
  margin-right: 5rem;
}
```

File: Week-1/Exercise1.1.md (hunk 4 above)
Comment (abhinavsingh06) [reply Week-1/Exercise1.1.md:1]: Added the image which explains the workflow for web browser.

**Pull request merged**